### DISCUSSION TITLE
1.5.2.2

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#595] [Обновления]
## Список изменений
- Фикс неправильного отображения кнопки в некоторых ситуациях (если выбрана нестандартная позиция кнопки)
- Фикс для работы в AdGuard

## What's Changed
* Исправление fetch при использовании AdGuard by @thedanill in https://github.com/ilyhalight/voice-over-translation/pull/594

## New Contributors
* @thedanill made their first contribution in https://github.com/ilyhalight/voice-over-translation/pull/594

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.2.1...1.5.2.2a

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.2.2a'>1.5.2.2</a>.</em>

**Comment by ilyhalight** [comment]
Перезалил релиз, теперь, должно нормально работать

**Reply by thedanill** [reply]
По крайней мере на macOS всё точно работает
<img width="500" alt="image" src="https://github.com/ilyhalight/voice-over-translation/assets/26379972/9e94d3c3-1284-47b7-b953-fee48e051779">
